src/com/codeenginestudio/management/book/util/HibernateUtil.java

```diff
@@ -0,0 +1,59 @@
+package com.codeenginestudio.management.book.util;
+
+import java.util.Properties;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.cfg.Environment;
+import org.hibernate.service.ServiceRegistry;
+
+import com.codeenginestudio.management.book.model.Author;
+import com.codeenginestudio.management.book.model.Book;
+import com.codeenginestudio.management.book.model.BookAndBookType;
+import com.codeenginestudio.management.book.model.BookType;
+
+public class HibernateUtil {
+
+	private static SessionFactory _sessionFactory;
+
+	public static SessionFactory _getSessionFactory() {
```

Comment:
@HuongNguyenThiThu @thuycao625 If you use the convention, you should use consistency across the project.
For example, you use the `_` prefix for a method or a private variable, so in a public method we don't use that prefix. It will make it easier for others to read and imagine the method.